bokeh/models/markers.py

```diff
@@ -40,7 +40,7 @@ class Marker(Glyph):
     The size (diameter) values for the markers in screen space units.
     """)
 
-    angle = NumberSpec(0, help="""
+    angle = AngleSpec(default=0.0, help="""
```

Comment:
I missed in the commit message that this changes from NumberSpec to AngleSpec, should have been its own commit really.